plugins/completion/cmp/sources/cmp-git.nix

```
@@ -7,18 +7,36 @@
 with lib;
 let
   cfg = config.plugins.cmp-git;
+
+  mkAction =
+    action: target:
+    helpers.defaultNullOpts.mkLuaFn "require('cmp_git.${action}').git.${target}" ''
+      Function used to ${action} the ${replaceStrings [ "_" ] [ " " ] target}.
+    '';
 in
 {
   options.plugins.cmp-git.settings = helpers.neovim-plugin.mkSettingsOption {
     pluginName = "cmp_git";
     options = {
-      filetypes = helpers.defaultNullOpts.mkListOf types.str ''["gitcommit" "octo"]'' ''
-        Filetypes for which to trigger.
-      '';
+      filetypes =
+        helpers.defaultNullOpts.mkListOf types.str
+          [
+            "gitcommit"
+            "octo"
+          ]
+          ''
+            Filetypes for which to trigger.
+          '';
```

```suggestion
          "Filetypes for which to trigger.";
```

This might cause nixfmt to attach the list brackets to the adjacent lines?

```nix
helpers.defaultNullOpts.mkListOf types.str [
            "gitcommit"
            "octo"
          ] "Filetypes for which to trigger.";
```

Even if not, I think its more readable for short descriptions like this?

---

Yes, I would prefer a "normal" string as well.

---

Done